audformat/core/utils.py

```diff
@@ -551,16 +561,21 @@ def intersect(
     _assert_index_alike(objs)
 
     # sort objects by length
-    objs = sorted(objs, key=lambda obj: len(obj))
-    # start from shortest index
-    index = objs[0]
-    mask = np.ones(len(index), dtype=bool)
-    for obj in objs[1:]:
-        mask &= index.isin(obj)
-        if not mask.any():
+    objs_sorted = sorted(objs, key=lambda obj: len(obj))
```

Comment:
We could immediately return with `return _alike_index(objs[0])` if `len(objs_sorted[0]) == 0`.

Reply:
Good idea, I added a short cut.